metrics/metrics.go

```diff
@@ -0,0 +1,119 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/metrics/sender"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+type Metrics struct {
```

Comment:
__[golint]__
* [golint] exported type Metrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->